pkg/controllers/sidecar/sync.go

```diff
@@ -2,41 +2,82 @@ package sidecar
 
 import (
 	"context"
-
-	"github.com/scylladb/scylla-operator/pkg/util/network"
+	"github.com/scylladb/scylla-operator/pkg/scyllaclient"
```

Comment:
please run `goimports` on every file you've changed.

Reply:
fixed